Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -21,15 +21,15 @@ class NetworkResponseHandler {
     private let LOG_TAG = "NetworkResponseHandler"
     private let serialQueue = DispatchQueue(label: "com.adobe.edge.eventsDictionary") // serial queue for atomic operations
     private let dataStore = NamedCollectionDataStore(name: EdgeConstants.EXTENSION_NAME)
-    private var updateLocationHint: (String?, TimeInterval?) -> Void
+    private var updateLocationHint: (String, TimeInterval) -> Void
```

Comment:
Let's add label to this
`(locationHint: String, ttl: TimeInterval) -> Void`

Reply:
Swift doesn't allow function types to have argument labels anymore.
https://github.com/apple/swift-evolution/blob/master/proposals/0111-remove-arg-label-type-significance.md

Reply:
What I can do is something like `private var updateLocationHint: (String, _ ttlSeconds: TimeInterval) -> Void`. We do something similar in Edge Identity.